docs/releases/ez_platform_v3.0_deprecations.md

```diff
@@ -80,15 +80,20 @@ Additionally, the `ez_trans_prop` Twig function has been removed.
 
 ### JavaScript event names and code cleanup
 
+#### Event names changed
+
 Selected event names have been changed.
 
-#### In Admin UI (`ezplatform-admin-ui`):
+##### In Admin UI (`ezplatform-admin-ui`):
 
 |Former name|New name|
 |-----------|--------|
 |`invalidFileSize`|`ez-invalid-file-size`|
 |`addressNotFound`|`ez-address-not-found`|
 |`cancelErrors`|`ez-cancel-errors`|
+|`userPreferedTimezone`|`userPreferredTimezone`|
```

Comment:
We should not put this here. `userPreferedTimezone`, `userPreferedFullDateTimeFormat`, `userPreferedShortDateTimeFormat` were not accessible from outside of the script. We can simply overlook these changes.